lib/evss/service.rb

```diff
@@ -6,6 +6,15 @@ module EVSS
   class Service < Common::Client::Base
     STATSD_KEY_PREFIX = 'api.evss'
 
+    def initialize(user)
```

Comment:
I like how much cleanup this allows within the specs - definitely will remember to avoid caching the service object outside of a request cycle